docs/server/source/quickstart.md

```diff
@@ -6,34 +6,55 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
-B. Run MongoDB. Open a Terminal and run the command:
+Default database directory where MongoDB stores its data: /data/db.
+
+B. To run MongoDB with default database path i.e. /data/db, open a Terminal and run the following command:
+```text
+$ mkdir -p /data/db
+```
+
+C. Assign rwx(read/write/execute) permissions to the user for default database directory:
+```text
+$ sudo chmod -R 700 /data/db
+```
+
+D. Run MongoDB. Open a Terminal and run the command:
```

Comment:
I guess we can remove the "Open a Terminal" bit now, since that was already covered in the new step B. How about this:

`D. Run MongoDB:`